### PR TITLE
#168977551 Configure Database

### PR DESCRIPTION
**Pull request by RusimbiP**
#### What does this PR do?
Have view database configured
#### Description of Task to be completed?
Have the following actions performable:
- Create Tables, 
- Drop tables,
- Insert initial data into the database
#### How should this be manually tested?
After cloning this project cd into it and run **npm i** to install dependencies

- Using SQL shell or pgAdmin create a database
- in .env file, create a DATABASE_URL following this format 
`postgres://{db_username}:{db_password}@{host}:{port}/{db_name}`

- run `npm run up` to check if tables can be created
- run `npm run seed` to check if data can be seeded into the database
- run `npm run down` to check if tables can be deleted/dropped
- run `npm run migrations` to check if tables can be deleted/dropped, created and have data inserted

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168977551 
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: server/v2/db/migrations/seed.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const pool = new Pool({ connectionString: conn });
+pool.on('connect', () => {
+  console.log('We connected to the database');
+});
+
+
+
+const seed = async () => {
+  const dataSeeds = [
+    'INSERT INTO users(firstname, lastname, email, password, gender, jobrole, department, address,  isAdmin) VALUES(\'John\', \'Doe\', \'taken@teamwork.com\',\'$2b$08$ByXvwUxxU47vy1tsKynZOecjXggYnsA4X9S3p2pWPKt1sRONu/P72\', \'Male\',\'Accountant\', \'Finance\', \'KG 232 Ave\', \'false\')',
```
Comment (Kasulejoseph) [inline server/v2/db/migrations/seed.js]: Move the admin users' details to the .env file, Such sensitive info should be exposed

Comment (RusimbiP) [reply server/v2/db/migrations/seed.js]: OK

File: server/v2/db/migrations/seed.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const pool = new Pool({ connectionString: conn });
+pool.on('connect', () => {
+  console.log('We connected to the database');
+});
+
+const adminFirstname = process.env.firstname;
+const adminLastname = process.env.lastname;
+const adminEmail = process.env.email;
+const pass = process.env.pass;
```
Comment (Kasulejoseph) [inline server/v2/db/migrations/seed.js]: Just destructure them from `process.env`

Comment (RusimbiP) [reply server/v2/db/migrations/seed.js]: Done!

**Pull request merged**